### PR TITLE
[BRD-1112] Extend rails 5 compatibility

### PR DESCRIPTION
**Pull request by valikos**
https://lumoslabs.atlassian.net/browse/BRD-1112

The gem was locked by rails `<= 5.0.7`. 
The latest security patch `5.0.7.2` was under support.

### CODE REVIEW

File: Gemfile-rails-5.0.x [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+gemspec
+
+
+group :development do
+  gem 'pry-byebug'
+end
+
+group :development, :test do
+  gem 'rails', '~> 5.0', '>= 5.0.7.2'
```
Comment (Jesterovskiy) [inline Gemfile-rails-5.0.x]: Maybe `<= 5.0.7.2`?

File: Gemfile-rails-5.1.x [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+gemspec
+
+
+group :development do
+  gem 'pry-byebug'
+end
+
+group :development, :test do
+  gem 'rails', '~> 5.1'
```
Comment (Jesterovskiy) [inline Gemfile-rails-5.1.x]: Maybe add limit `< 5.2`?

File: Gemfile-rails-5.2.x [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+gemspec
+
+
+group :development do
+  gem 'pry-byebug'
+end
+
+group :development, :test do
+  gem 'rails', '~> 5.2', '>= 5.2.4.3'
```
Comment (Jesterovskiy) [inline Gemfile-rails-5.2.x]: `< 6.0`?

**Review by Jesterovskiy — APPROVED** [review]
👍

File: Gemfile-rails-5.1.x (hunk 2 above)
Comment (valikos) [reply Gemfile-rails-5.1.x]: Yeah, make sense. Will change. Thanks.

**Review by Groovej — APPROVED** [review]
Nice.

**Review by camertron — APPROVED** [review]
😎

**Pull request merged**